packages/helpwave_widget/lib/src/static_progress_indicator/static_progress_indicator.dart

```diff
@@ -0,0 +1,54 @@
+import 'package:flutter/material.dart';
+import 'package:helpwave_theme/constants.dart';
+
+/// A Circular Progress indicator
+///
+/// Example:
+/// ```dart
+/// StaticProgressIndicator(
+///   progress: 0.2,
+/// )
+/// ```
+class StaticProgressIndicator extends StatelessWidget {
+  final Size size;
+
+  final double strokeWidth;
+
+  /// The semanticsLabel for Screen Reading Software
+  final String? semanticsLabel;
+
+  /// Progress as a Percentage between 0.0 to 1.0
+  final double progress;
+
+  /// [Color] that indicates Progress
+  final Color? color;
+
+  /// [Color] that is the background of the ring
+  final Color? backgroundColor;
+
+  const StaticProgressIndicator({
+    super.key,
+    required this.progress,
+    this.size = const Size.square(iconSizeSmall),
+    this.strokeWidth = 8,
+    this.semanticsLabel,
+    this.color,
+    this.backgroundColor,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    return SizedBox(
+      height: size.height - strokeWidth,
+      width: size.width - strokeWidth,
+      child: CircularProgressIndicator(
+        strokeWidth: strokeWidth,
+        semanticsLabel: semanticsLabel,
+        color: color ?? Theme.of(context).colorScheme.onBackground,
+        backgroundColor:
+            backgroundColor ?? Theme.of(context).colorScheme.background,
+        value: progress,
+      ),
```

Comment:
Rotate the progress indicator by 90 degrees. We need it to start at 3 o'clock "loading" counter clockwise.

Reply:
Why would we want to start at 3 o'clock and go backwards? 
Only going backwards maybe, that's not completly uncommon, but starting at 3 o'clock doesn't make much sense to me, when the user only wants to quickly parse how much progress has been made or do we intent to make the user pause to register and parse the component?

Reply:
Look at the screen designs. It's defined...
This is not the place to discuss design specific patterns.